boards/arm/nucleo_l432kc/doc/nucleol432kc.rst

```diff
@@ -0,0 +1,226 @@
+.. _nucleo_l432kc_board:
+
+ST Nucleo L432KC
+################
+
+Overview
+********
+
+The Nucleo L432KC board features an ARM Cortex-M4 based STM32L432KC MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the Nucleo L432KC board:
+
+
+- STM32 microcontroller in UFQFPN32 package
+- Arduino Uno V3 connectivity
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector
+- Flexible board power supply:
+       - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+       - Power management access point
+- Three LEDs: USB communication (LD1), user LED (LD2), power LED (LD3)
+- One push-button: RESET
+
+.. image:: img/nucleo32_ulp_logo.jpg
+     :width: 250px
+     :align: center
+     :height: 188px
+     :alt: Nucleo L432KC
+
+More information about the board can be found at the `Nucleo L432KC website`_.
+
+Hardware
+********
+
+The STM32L432KC SoC provides the following hardware IPs:
+
+- Ultra-low-power with FlexPowerControl (down to 28 nA Standby mode and 84 μA/MHz run mode)
+- Core: ARM® 32-bit Cortex®-M4 CPU with FPU, frequency up to 80 MHz, 100DMIPS/1.25DMIPS/MHz (Dhrystone 2.1)
+- Clock Sources:
+        - 32 kHz crystal oscillator for RTC (LSE)
```

Comment:
Lists (and sub-lists) need a blank line before and after the lists, so add a blank line before - 32 kHz